src/main/java/ticTacToe/game/GameFlow.java

```diff
@@ -17,12 +18,12 @@
     private Player firstPlayer;
     private Player secondPlayer;
 
-    public GameFlow(Ui ui, Grid grid, Lines lines, PlayersFactory playersFactory) {
+    public GameFlow(Ui ui, Lines lines, PlayersFactory playersFactory) {
         this.ui = ui;
-        this.grid = grid;
+        ui.welcomePlayer();
```

Comment:
Its usually discouraged to have 'business logic' in the constructor. It can be overlooked and forgotten about, as constructors usually just create an object.
Can this welcome message be moved into 'runGame()'?